rotkehlchen/db/history_events.py

```diff
@@ -802,3 +803,50 @@ def edit_event_extra_data(
             (json.dumps(extra_data), event.identifier),
         )
         event.extra_data = extra_data
+
+    def query_wrap_stats(self, from_ts: Timestamp, to_ts: Timestamp) -> dict[str, Any]:
+        """Query simple statistics about the user to show them as a wrap for the year"""
+        from_ts_ms, to_ts_ms = ts_sec_to_ms(from_ts), ts_sec_to_ms(to_ts)
+        with self.db.conn.read_ctx() as cursor:
+            cursor.execute(
+                'SELECT SUM(CAST(amount AS FLOAT)) FROM history_events JOIN evm_events_info '
+                'ON history_events.identifier=evm_events_info.identifier WHERE '
+                "asset='ETH' AND type='spend' and subtype='fee' AND counterparty='gas' AND "
+                'timestamp >= ? AND timestamp <= ?',
+                (from_ts_ms, to_ts_ms),
+            )
+            eth_on_gas = str(cursor.fetchone()[0])
+            cursor.execute(
+                'SELECT chain_id, COUNT(*) FROM evm_transactions '
+                'WHERE timestamp >= ? AND timestamp <= ? GROUP BY chain_id',
+                (from_ts, to_ts),
+            )
+            transactions_per_chain = {ChainID.deserialize_from_db(row[0]).name: row[1] for row in cursor}  # noqa: E501
+            cursor.execute(
+                'SELECT location, COUNT(*) from trades '
+                'WHERE timestamp >= ? AND timestamp <= ? GROUP BY location',
+                (from_ts, to_ts),
+            )
+            trades_by_exchange = {str(Location.deserialize_from_db(row[0])): row[1] for row in cursor}  # noqa: E501
+            cursor.execute(
```

Comment:
this is only for users with premium + gnosis pay right? So perhaps if there is nothing return empty? How should this work?

Reply:
if there is not information it will be empty